### DISCUSSION TITLE
Translator application for Portuguese

### DISCUSSION THREAD

**Discussion by fupicat** [ScratchAddons/ScratchAddons#1027] [Discussion]
My Transifex username is [Fupicat](https://www.transifex.com/user/profile/fupicat/), I'd like to translate Scratch Addons into Portuguese and Brazilian Portuguese.
I'm part of the localization team for Scratch and I also translated TurboWarp, if that's important.

**Comment by UltraCupcake** [comment]
Huh... I was going to translate Portuguese (pt)

**Reply by fupicat** [reply]
We could probably do it together if there's a lot of stuff to translate.

**Reply by UltraCupcake** [reply]
Haha yeah why not? But I think that Brazilian Portuguese and Portuguese (pt) have some differences...

**Reply by fupicat** [reply]
I've heard that the Scratch translation for european Portuguese is severely outdated, because it doesn't conform with the 1990 reform. Reading Portuguese media and dictionaries I see pretty much no difference nowadays.

**Reply by fupicat** [reply]
Though if you are a native Portuguese speaker I don't see why you wouldn't be more qualified than me to translate into your own language, I just offered it because I thought no one else would apply lol

**Reply by WorldLanguages** [reply]
Keep in mind it's unlikely we'd have two variations of portuguese, in the same way we have a single local for Spanish even tho the Scratch editor has two.

**Reply by fupicat** [reply]
Ok that's fair.
